commercial/app/views/books/booksStandardV2.scala.html

```diff
@@ -2,7 +2,7 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
-@containerWrapper {
+@containerWrapper(Seq(), "")(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty) {
```

Comment:
You can use `Nil` for an empty `Seq` but I suppose this is going to be filled in eventually anyway.